src/components/utdragFraSykefravaeret/Sykmeldinger.tsx

```diff
@@ -0,0 +1,182 @@
+        </ExpansionCard.Title>
+      </StyledExpantionCardHeader>
+      <ExpansionCard.Content>
+        <SykmeldingUtdragFraSykefravaretVisning sykmelding={sykmelding} />
+      </ExpansionCard.Content>
+    </ExpansionCard>
+  );
+};
+
+interface UtvidbarTittelProps {
+  sykmelding: SykmeldingOldFormat;
+}
+
+const SykmeldingTittelbeskrivelse = ({ sykmelding }: UtvidbarTittelProps) => {
+  const sykmeldingPerioderSortertEtterDato =
+    sykmeldingperioderSortertEldstTilNyest(
+      sykmelding.mulighetForArbeid.perioder
+    );
+
+  const periode = `${tilLesbarPeriodeMedArstall(
+    tidligsteFom(sykmelding.mulighetForArbeid.perioder),
+    senesteTom(sykmelding.mulighetForArbeid.perioder)
+  )}: `;
+  const graderinger = stringMedAlleGraderingerFraSykmeldingPerioder(
+    sykmeldingPerioderSortertEtterDato
+  );
+  const diagnose = `${sykmelding.diagnose.hoveddiagnose?.diagnosekode} (${sykmelding.diagnose.hoveddiagnose?.diagnose})`;
+  const erViktigInformasjon = erEkstraInformasjonISykmeldingen(sykmelding);
+  const sykmelder = sykmelding.bekreftelse.sykmelder;
+  const arbeidsgiver = arbeidsgivernavnEllerArbeidssituasjon(sykmelding);
+  const erIkkeTattIBruk = sykmelding.status === SykmeldingStatus.NY;
+  const erUtenArbeidsgiver = erSykmeldingUtenArbeidsgiver(sykmelding);
+
+  return (
+    <div className="w-full flex flex-col">
+      <div className="flex justify-between mb-2">
+        <div>
+          {periode}
+          {graderinger}
+        </div>
+        <div className="flex gap-4">
+          {erIkkeTattIBruk && (
+            <Tag variant="warning" size="small">
+              {texts.ny}
+            </Tag>
+          )}
+          {erUtenArbeidsgiver && (
+            <Tag variant="info" size="small">
+              {texts.utenArbeidsgiver}
+            </Tag>
+          )}
+          {erViktigInformasjon && (
+            <img
+              height={18}
+              alt="Viktig informasjon"
+              src={MerInformasjonImage}
+            />
+          )}
+        </div>
+      </div>
+      {sykmelding.diagnose.hoveddiagnose && (
+        <Info label={"Diagnose: "} text={diagnose} />
+      )}
+      {sykmelder && <Info label={"Sykmelder: "} text={sykmelder} />}
+      {arbeidsgiver && <Info label={"Arbeidsgiver: "} text={arbeidsgiver} />}
+      {sykmelding.yrkesbetegnelse && (
+        <Info
+          label={"Stilling fra sykmelding: "}
+          text={sykmelding.yrkesbetegnelse}
+        />
+      )}
+      {sykmelding.papirsykmelding && <PapirsykmeldingTag />}
+    </div>
+  );
+};
+
+export default function Sykmeldinger() {
+  const { sykmeldinger } = useSykmeldingerQuery();
+  const { latestOppfolgingstilfelle } = useOppfolgingstilfellePersonQuery();
+
+  const aktuelleSykmeldinger = sykmeldinger.filter(
+    (sykmelding) =>
+      sykmelding.status === SykmeldingStatus.SENDT ||
+      sykmelding.status === SykmeldingStatus.NY ||
+      erSykmeldingUtenArbeidsgiver(sykmelding)
+  );
```

Comment:
De to første sjekkene er for å få med de sykmeldingene med arbeidsgiver og den siste er for de uten arbeidsgiver.